tgbotapi.extensions.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/chat/modify/UnpinChatMessage.kt

```diff
@@ -3,12 +3,16 @@ package dev.inmo.tgbotapi.extensions.api.chat.modify
 import dev.inmo.tgbotapi.bot.TelegramBot
 import dev.inmo.tgbotapi.requests.chat.modify.UnpinChatMessage
 import dev.inmo.tgbotapi.types.ChatIdentifier
+import dev.inmo.tgbotapi.types.MessageIdentifier
 import dev.inmo.tgbotapi.types.chat.abstracts.PublicChat
+import dev.inmo.tgbotapi.types.message.abstracts.Message
 
 suspend fun TelegramBot.unpinChatMessage(
-    chatId: ChatIdentifier
-) = execute(UnpinChatMessage(chatId))
+    chatId: ChatIdentifier,
+    messageId: MessageIdentifier? = null
+) = execute(UnpinChatMessage(chatId, messageId))
 
 suspend fun TelegramBot.unpinChatMessage(
-    chat: PublicChat
-) = unpinChatMessage(chat.id)
+    chat: PublicChat,
```

Comment:
I think, that implementation of method with `message` must not use only one parameter `message`. Besides, we must keep extension with chat and messageId

Reply:
Could you please clarify what's the issue here?

Reply:
I meant:

```kotlin
suspend fun TelegramBot.unpinChatMessage(
    chat: Chat,
    messageId: MessageIdentifier? = null
) = unpinChatMessage(chat.id, messageId)

suspend fun TelegramBot.unpinChatMessage(
    message: Message
) = unpinChatMessage(message.chat.id, message.messageId)
```

Reply:
What about the newest changes?